### DISCUSSION TITLE
How to join the weekly meetings? Where find recording?

### DISCUSSION THREAD

**Discussion by porcelli** [apache/incubator-kie-issues#906] [Weekly Meetings]
Join Zoom Meeting
https://us06web.zoom.us/j/74923352785?pwd=5cbLmE94OllYTPgxr4j2qa1hu5AG0b.1

Meeting ID: 749 2335 2785
Passcode: 253317

Link for the calendar [here](https://us04web.zoom.us/meeting/upAkdeqrqzktEtF_15tzK_48bqCYXWFrQ3-C/ics?icsToken=98tyKu6spzkpGdGXtxGCR7YIHYjob-nwmHpaj7d7o06wFzp4XVeuNPNhC4pINsrh) 

You can follow the topics and discussions in this [discussion category](https://github.com/apache/incubator-kie-issues/discussions/categories/weekly-meetings).

In case recording is not available anymore, you can download the recording from [here](https://drive.google.com/drive/folders/1GV8wQRfIbAMgRmKHZwXYwD9g9Fhj8rfQ)

**Comment by mareknovotny** [comment]
are the recordings somewhere shared? the [link](https://drive.google.com/drive/folders/1GV8wQRfIbAMgRmKHZwXYwD9g9Fhj8rfQ) above doesn't show all recordings from past weeks/months?

**Comment by bmanojprabhakar** [comment]
@porcelli Do we have any upcoming schedule? Seems the calendar invite is invalid, unable to import

**Comment by porcelli** [comment]
@bmanojprabhakar meetings are happening every friday 9am EST
